snapcraft/internal/parser.py

```diff
@@ -233,6 +233,22 @@ def _process_wiki_entry(entry, master_parts_list):
         master_parts_list.update(parts_list)
 
 
+def _handle_wiki_entry(entry, master_parts_list):
```

Comment:
you have handle_wiki_entry and process_wiki_entry. Just with the name it will be impossible to understand what each one does. I can't think of better names, so it would be good to put docstrings.